dotnet3.5/STS/AssumeRole/AssumeRoleExampleTest/AssumeRoleTests.cs

```diff
@@ -0,0 +1,100 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+using Amazon;
+using Amazon.SecurityToken;
+using Amazon.SecurityToken.Model;
+using Moq;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace AssumeRoleExampleTest
+{
+    public class AssumeRoleTests
+    {
+        private string roleArnToAssume = "arn:aws:iam::123456789012:role/testAssumeRole";
+        private static readonly RegionEndpoint REGION = RegionEndpoint.USWest2;
+
+        [Fact]
+        public async Task GetCallerIdentityAsyncTest()
+        {
+            // Create the mock SecurityToken client.
+            // Define variable
```

Comment:
Comments should explain the code or point out quirks, not just restate it. (In other words, I'd remove this comment because I don't think it adds value.)

Reply:
Removed.